deploy/rp-development.json

```diff
@@ -2,6 +2,12 @@
     "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
     "contentVersion": "1.0.0.0",
     "parameters": {
+        "adminApiCaBundle": {
+            "type": "string"
+        },
+        "adminApiClientCertCommonName": {
+            "type": "string"
+        },
```

Comment:
We don't honour the admin API authentication settings in dev, therefore I don't think rp-development.json should change, or prepare-a-shared-rp-development-environment.md, or deploy-shared-env.sh?

Reply:
Right! We don't even deploy vmss in shered env.